src/main/scala/org/constellation/DAO.scala

```diff
@@ -113,6 +113,12 @@ class DAO() extends NodeData with EdgeDAO with SimpleWalletLike with StrictLoggi
 
     implicit val ioTimer: Timer[IO] = IO.timer(ConstellationExecutionContext.unbounded)
 
+    if (ConfigUtil.isEnabledAwsStorage) {
+      cloudStorage = new AwsStorage[IO]
+    } else if (ConfigUtil.isEnabledGcpStorage) {
+      cloudStorage = new GcpStorage[IO]
+    }
+
```

Comment:
At first glance looks like a code smell because technically I can set both `isEnabledGcpStorage` and `isEnabldAwsStorage` to `true`. Then the priorities of choice are non-deterministic (Aws has priority but you need to dive into the code).

The best way would be to just use enumeration like:

```
ConfigUtil.storageProvider
```

and just give options like `Local`, `AWS`, `GCP`.

Also keep in mind that such abbreviations like `Aws` or `Gcp` should be `AWS` and `GCP` but it is just the detail and common issue with `camelCase`/`PascalCase`

Reply:
it is specially made, in the future we will support any number of cloud storages, 
and where you turn it true, it will upload files there, but I didn't have time to turn it into a cloud storages list and handle it on services

Reply:
+1 for AWS/GCP